### PR TITLE
Script for checking PR state

### PR DESCRIPTION
**Pull request by nebril**
`contrib/scripts/checkpr.sh PR_ID` plays an alert when your PR build
finishes.

### CODE REVIEW

File: contrib/scripts/checkpr.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+until [ "$result" != "null" ]; do
+	sleep 60
+	result=$(curl -s https://jenkins.cilium.io/job/cilium/job/cilium/job/PR-${PR_ID}/lastBuild/api/json | jq ".result")
+done
+
+name=$(uname)
+for run in {1..5}
+do
+	if [ "$name" == "Linux" ]; then
+		paplay /usr/share/sounds/gnome/default/alerts/bark.ogg
```
Comment (joestringer) [inline contrib/scripts/checkpr.sh]: Would you mind wrapping this with a `if [ -e $BARK_PATH ]; then ... else ...` and make the else statement fall back to something that should be more widely available, perhaps like `paplay /usr/share/sounds/freedesktop/stereo/bell.oga`?

File: contrib/scripts/checkpr.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```
Comment (joestringer) [inline contrib/scripts/checkpr.sh:1]: Could probably add a simple comment below that the script requires curl, jq, and paplay (if on Linux). Could do it as a comment here, or alternatively an actual check something like:

```
if ! which curl >/dev/null || ! which jq >/dev/null; then
    echo "This tool requires 'curl' and 'jq' utilities to work correctly. Please install them."
    exit 1
fi
```

**Review by joestringer — COMMENTED** [review]
Looks good, some minor comments. Would like someone with osx to review to make sure it works for them too.

**Comment by nebril** [comment]
@joestringer thanks for your review, addressed your comments.

**Review by aalemayhu — CHANGES_REQUESTED** [review]
Please use a sound file on macOS as well, like below

```diff
diff --git a/contrib/scripts/checkpr.sh b/contrib/scripts/checkpr.sh
index 63e0b1772cda..7a29c3ce6faf 100755
--- a/contrib/scripts/checkpr.sh
+++ b/contrib/scripts/checkpr.sh
@@ -29,6 +29,6 @@ do
                paplay ${BARK_PATH}
        else
                # for osx
-               say "bark"
+               afplay /System/Library/Sounds/Ping.aiff
        fi
 done
```

**Review by joestringer — APPROVED** [review]
LGTM

**Comment by nebril** [comment]
@scanf done, please re-review

**Review by aalemayhu — APPROVED** [review]
LGTM, thanks.

File: contrib/scripts/checkpr.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	exit
+fi
+
+PR_ID=$1
+
+result="null"
+
+until [ "$result" != "null" ]; do
+	sleep 60
+	result=$(curl -s https://jenkins.cilium.io/job/cilium/job/cilium/job/PR-${PR_ID}/lastBuild/api/json | jq ".result")
```
Comment (aanm) [inline contrib/scripts/checkpr.sh:20]: Will it still works with the new ginkgo jenkins jobs?

Comment (aalemayhu) [reply contrib/scripts/checkpr.sh:20]: Probably not since the job name will be different. The URL can easily be updated later. Should not block this.

**Comment by joestringer** [comment]
test-me-please

**Comment by joestringer** [comment]
@nebril Are you waiting on something to merge this PR?

**Pull request merged**